### DISCUSSION TITLE
Free Storage Limit Reached - Notice Email

### DISCUSSION THREAD

**Discussion by ryanbuening** [ravendb/ravendb#15299] [General]
I received an email that stated: `Your free cloud instance has reached 90% of its maximum storage capacity.`

I logged into the Studio and the Storage pane reports the following.

![image](https://user-images.githubusercontent.com/812974/202861952-c6ef8378-a365-40b4-a48c-9fc4e70d6bd3.png)

How is the 4.9 GB used being calculated? I'm not seeing where I am at 90% capacity.

**Comment by garayx** [comment]
Hi, please submit a ticket using the support form, on [cloud portal](https://cloud.ravendb.net/portal), so we can take a look on your instance.

**Comment by ayende** [comment]
To answer the actual question, the 10GB also include the OS itself, RavenDB, etc.
That accounts for most of the initial space.

It is possible that the system used more disk space for a short time, triggering the notification. 
* Updates / patching the OS
* Long transactions / big imports on RavenDB, etc

**Comment by ryanbuening** [comment]
Thanks for the info. I submitted a ticket and this is resolved.
